src/config/theme.js

```diff
@@ -281,6 +281,35 @@ export const theme = {
         },
       },
     },
+    MuiInput: {
+      root: {
+        fontSize: px(smallText),
+        lineHeight: lineHeight24,
+        padding: `8px ${px(space2X)}`,
+        color: textColorDarkGrey,
+      },
+      input: {
+        padding: 0,
+      },
+      underline: {
+        '&:after': {
+          borderBottom: `2px solid ${primaryBrandColor}`,
+        },
+      },
+      error: {
+        '&:after': {
+          borderBottomColor: `${supportColorRed100} !important`,
+        },
+      },
+    },
+    MuiFormHelperText: {
```

Comment:
does this apply to all `MuiInput`s? or do we have to wrap all input fields with `FormHelper`?

Reply:
The TextField contains a FormHelperText. If use Input, it needs FormHelperText. In this project, using TextField with FormHelperText, it doesn't matter.